Orchestrator/Prognetics.Saga.Orchestrator/Prognetics.Saga.Orchestrator.csproj

```diff
@@ -28,6 +28,10 @@
     </None>
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="MongoDB.Driver" Version="2.20.0" />
+  </ItemGroup>
```

Comment:
This dependency shouldn't be in the Orchestrator. I believe that it should have no knowledge of the Log implementation and it should be abstracted away.

Reply:
Ok I am  going to move the implementation to the seperate dll. I understood the description of the issue wrong then: https://github.com/orgs/Prognetics/projects/1?pane=issue&itemId=19711249